CustomUserOperationEventListener/src/main/java/org/wso2/carbon/sample/user/operation/event/listener/CustomUserOperationEventListener.java

```diff
@@ -0,0 +1,66 @@
+package org.wso2.carbon.sample.user.operation.event.listener;
+
+import org.apache.commons.lang.StringUtils;
+import org.wso2.carbon.user.core.UserStoreException;
+import org.wso2.carbon.user.core.UserStoreManager;
+import org.wso2.carbon.user.core.common.AbstractUserOperationEventListener;
+
+import java.security.SecureRandom;
+import java.util.Map;
+import java.util.Random;
+
+public class CustomUserOperationEventListener extends AbstractUserOperationEventListener {
```

Comment:
Shall we use a descriptive name instead of ```CustomUserOperationEventListener```?
Something like ```RandomPasswordGenerator``` or ```RandomPasswordUserOperationEventListener```